### PR TITLE
AssetTree styles props

### PR DESCRIPTION
**Pull request by joyrostie**
Styles props for AssetTree component
Storybook readme
Tests for applying inline styles

### CODE REVIEW

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-289.gearboxjs.preview.cogniteapp.com

**Comment by joyrostie** [comment]
> Could you also add theming support for this component as @xitman78 did for TenantSelector?

Yes, I will check it on TenantSelector

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-289.gearboxjs.preview.cogniteapp.com

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-289.gearboxjs.preview.cogniteapp.com

File: src/components/ThemeProvider/stories/ThemeProvider.stories.tsx [hunk 1]
```diff
@@ -34,30 +36,50 @@ storiesOf('ThemeProvider', module).add(
...
+        content: tenantSelector,
+      },
+    }
+  )
+  .add(
+    'Asset Tree',
+    () => (
+      <ThemeProvider
+        theme={{
+          textFamily: 'Consolas',
```
Comment (xitman78) [inline src/components/ThemeProvider/stories/ThemeProvider.stories.tsx]: Change it for some web safe font.

File: src/components/ThemeProvider/stories/full.md [hunk 2]
```diff
@@ -43,20 +43,19 @@ function ExampleComponent(props) {
 
 | Property           | Description                                  | Type                                                           | Default |
 | ------------------ | -------------------------------------------- | -------------------------------------------------------------- | ------- |
-| `theme`            | A custom theme                               | `GearboxTheme`                                                 |         |
+| `theme`            | A custom theme                               | `GearboxTheme`, `GearboxThemeOptional`                         |         |
```
Comment (xitman78) [inline src/components/ThemeProvider/stories/full.md]: Leave it as it was before just `GearboxTheme`

File: src/components/ThemeProvider/stories/full.md [hunk 3]
```diff
@@ -43,20 +43,19 @@ function ExampleComponent(props) {
...
 | `children`         | A child component wrapped with the theme     | `React.ReactChild`                                             |         |
 
 
 ### Types
 
-#### GearboxTheme
+#### GearboxTheme & GearboxThemeOptional
```
Comment (xitman78) [inline src/components/ThemeProvider/stories/full.md]: Leave it as it was before

File: src/mocks/assetsList.ts [hunk 4]
```diff
@@ -88,6 +88,14 @@ export const SearchValue: AdvancedSearch = {
 
 export const assetsList = [SKA, IAA, VAL];
 
+export const ASSET_TREE_STYLES: AssetTreeStyles = {
+  list: {
+    fontFamily: 'Consolas',
```
Comment (xitman78) [inline src/mocks/assetsList.ts]: web safe font

File: src/components/ThemeProvider/stories/ThemeProvider.stories.tsx (hunk 1 above)
Comment (joyrostie) [reply src/components/ThemeProvider/stories/ThemeProvider.stories.tsx]: Font replaced.

File: src/components/ThemeProvider/stories/full.md (hunk 2 above)
Comment (joyrostie) [reply src/components/ThemeProvider/stories/full.md]: Removed

File: src/components/ThemeProvider/stories/full.md (hunk 3 above)
Comment (joyrostie) [reply src/components/ThemeProvider/stories/full.md]: Removed

File: src/mocks/assetsList.ts (hunk 4 above)
Comment (joyrostie) [reply src/mocks/assetsList.ts]: Font replaced.

File: src/components/ThemeProvider/stories/full.md [hunk 5]
```diff
@@ -51,12 +51,11 @@ function ExampleComponent(props) {
 
 #### GearboxTheme
 
-The object that defines colours and some other CSS properties for `@cognite/gearbox` components.
+The objects that define colours and some other CSS properties for `@cognite/gearbox` components.
```
Comment (xitman78) [inline src/components/ThemeProvider/stories/full.md]: Why the plural form?

Comment (joyrostie) [reply src/components/ThemeProvider/stories/full.md]: my bad, was left from two types overview. fixed

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-289.gearboxjs.preview.cogniteapp.com

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-289.gearboxjs.preview.cogniteapp.com

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-289.gearboxjs.preview.cogniteapp.com

**Pull request merged**